src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -143,7 +150,55 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     if (onTabSelected) {
       onTabSelected(index);
     }
-    setCurrentIndex(index);
+    setCurrentActiveIndex(index);
+  };
+
+  const onKeyDown = (event: React.KeyboardEvent<HTMLUListElement>) => {
+    // alt + left/right is used by browser / assistive technologies
+    if (event.altKey) {
+      return;
+    }
+
+    const isTabDisabled = (index: number) => {
```

Comment:
I would prefer this out, than function in function in function...

Reply:
I like keeping small helper functions confined to the scope they are used in.

We are doing it the same way in Menu:
https://github.com/iTwin/iTwinUI-react/blob/8f3e071c95016dd7722c5b3767460d647c2ccf04/src/core/Menu/Menu.tsx#L58-L59